korma/src/commonMain/kotlin/com/soywiz/korma/geom/Point.kt

```diff
@@ -321,14 +321,15 @@ data class Point(
     fun changeLength(length: Float): Point = changeLength(length.toDouble())
     fun changeLength(length: Int): Point = changeLength(length.toDouble())
     fun changeLength(length: Double): Point {
+        if (this.length == 0.0 || this.length == length) return this
         this.normalize()
         this.setTo(this.x * length, this.y * length)
         return this
     }
 
     val angle: Angle
         get() {
-            var angle: Double = atan2(this.y, this.x) * RAD2DEG
+            var angle = atan2(this.y, this.x).degrees
             if (angle < 0) angle += 360.0
             return angle.degrees
         }
```

Comment:
Actually, you can just do:

```kotlin
return Angle.atan2(this.y, this.x)
```
or
```kotlin
return Angle.between(0.0, 0.0, this.x, this.y).normalized // Normalizing is converting any angle into the right 0..360 degrees
```


Why is the x and y in opposite places, and why are you normalizing the angle later? Is this what people would expect? Shouldn't it be something like `normalizedAngle`? Or just provide the angle and let people normalize it if they need it?